en/changelog.md

```diff
@@ -1,3 +1,30 @@
+## v5.4.1
+
++ [Feature] [i18n]: add Hungarian(hu_HU) translation [#17978](https://github.com/apache/echarts/issues/17978) ([ifjkovacsik](https://github.com/ifjkovacsik))
++ [Feature] [type]: export `ECElementEvent` for TypeScript [#17892](https://github.com/apache/echarts/issues/17892) ([keita-determined](https://github.com/keita-determined))
++ [Feature] [svg]: support gradients and patterns for background color in SVG renderer [#959](https://github.com/ecomfe/zrender/issues/959) ([plainheart](https://github.com/plainheart))
++ [Feature] [candlestick]: provide borderColorDoji option for custom doji color [#17717](https://github.com/apache/echarts/issues/17717) ([Ovilia](https://github.com/Ovilia))
++ [Feature] [sankey]: `sankey` support `edgeLabel` [#16767](https://github.com/apache/echarts/issues/16767) ([susiwen8](https://github.com/susiwen8))
++ [Feature] [graph]: make graph node draggable in `none` and `circular` layout [#15428](https://github.com/apache/echarts/issues/15428) ([kongmoumou](https://github.com/kongmoumou))
++ [Fix] [dataZoom]: fallback to extent start/end when value or percent is invalid [#17934](https://github.com/apache/echarts/issues/17934) ([plainheart](https://github.com/plainheart))
++ [Fix] [pie]: fix `center` may not be converted to correct point when coordinate system is specified [#17920](https://github.com/apache/echarts/issues/17920) ([plainheart](https://github.com/plainheart))
++ [Fix] [type]: add `string` & `number` type into the `center` option of pie series [#17917](https://github.com/apache/echarts/issues/17917) ([plainheart](https://github.com/plainheart))
++ [Fix] [heatmap]: fix heatmap label may overlap when `dataZoom` is enabled [#17876](https://github.com/apache/echarts/issues/17876) ([plainheart](https://github.com/plainheart))
++ [Fix] [tooltip]: fix tooltip arrow may cover the content when border is wide [#17875](https://github.com/apache/echarts/issues/17875) ([plainheart](https://github.com/plainheart))
++ [Fix] [i18n]: language code for Ukrainian is UK, not UA (follow-up) [#17812](https://github.com/apache/echarts/issues/17812) ([striezel](https://github.com/striezel))
++ [Fix] [type]: add overflow property to AxisLabelBaseOption interface. close #17363 [#17808](https://github.com/apache/echarts/issues/17808) ([breizh24](https://github.com/breizh24))
++ [Fix] [lines)]: add missing `function` callback into `lineStyle.color` for lines series. [#17780](https://github.com/apache/echarts/issues/17780) ([plainheart](https://github.com/plainheart))

```

Comment:
The scope of this change should be `type` rather than `lines` and please put this item below.